src/object/cli_ec.c

```diff
@@ -57,27 +57,67 @@ struct ec_params {
 	struct ec_params        *next;	/* Pointer to next entry in list. */
 };
 
+struct ec_fetch_params {
+	daos_iod_t		*iods;	/* Replaces iod array in fetch. */
+	unsigned int		 nr;	/* number of records in iods    */
+	daos_iod_t			 niod;
+	struct ec_fetch_params		*next;/* Next entry in list */
```

Comment:
it might just be github, but alignment looks off to me

Reply:
Fixed.